src/main/java/de/komoot/photon/RequestHandler.java

```diff
@@ -59,10 +59,13 @@ public String handle(Request request, Response response) {
 			limit = 15;
 		}
 
-		List<JSONObject> results = searcher.search(query, lang, lon, lat, limit, true);
+        String tagKey = request.queryParams("tagKey");
+        String tagValue = request.queryParams("tagValue");
```

Comment:
Very good point. In the mappings.json, it defines ```osm_key``` and ```osm_value```. Did you really mean to use ```tag_key``` and ```tag_value```? If I don't hear back, I will change to ```osm_key``` and ```osm_value```